src/libAtomVM/nifs.c

```diff
@@ -22,21 +22,12 @@
 #define _GNU_SOURCE
 #endif
 
-// used to be required before C11 for PRI* format macros
-#ifndef __STDC_FORMAT_MACROS
-#define __STDC_FORMAT_MACROS
-#endif
-
-// stdio.h must be included before of inttypes.h to avoid a bug in newlib (observed on RPi 2040)
-// that makes PRIu64 unavailable
-#include <stdint.h>
-#include <stdio.h>
-
-#include <inttypes.h>
+#include "nifs.h"
```

Comment:
When you have `foo.c`, it is a good practice to include `foo.h` as first thing. This will make possible to spot some kind of errors like missing includes for used types in the header files etc...
If you move it later you never know if your .h file has all the right includes or if it is just inheriting definitions from the headers included before of it.

Reply:
Yes. You are correct, I think I wrote that on my second pass through the file, and forgot that this file was nifs.c.